content/en/logs/troubleshooting/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Logs Troubleshooting
+kind: documentation
+---
+
+If you experience unexpected behavior with Datadog Logs, there are a few common issues you can investigate and this guide may help resolve issues quickly. If you continue to have trouble, reach out to [Datadog support][1] for further assistance.
+
+## Why are my logs missing today?
+
+You have not made any changes to your log configuration but [Logs Explorer][2] shows that logs are missing today. This may be happening because you have reached your daily quota.
```

Comment:
```suggestion
You have not made any changes to your log configuration, but the [Logs Explorer][2] shows that logs are missing for today. This may be happening because you have reached your daily quota.
```